packages/gatsby-plugin-mdx/utils/get-source-plugins-as-remark-plugins.js

```diff
@@ -20,66 +19,56 @@ module.exports = async function getSourcePluginsAsRemarkPlugins({
   ...helpers
 }) {
   debug(`getSourcePluginsAsRemarkPlugins`)
-  let pathPlugin = undefined
+
+  let userPlugins = []
```

Comment:
can't this be const?
```suggestion
  const userPlugins = []
```

Reply:
Ah yes